container/service_options.go

```diff
@@ -101,11 +107,15 @@ func (options *ServiceOptions) swarmMounts(force bool) []mount.Mount {
 // Each network will be attached based on their networkID but also based on aliases
 // These aliases will make services accessible from other containers inside the same network
 func (options *ServiceOptions) swarmNetworks() (networks []swarm.NetworkAttachmentConfig) {
-	networks = make([]swarm.NetworkAttachmentConfig, len(options.NetworksID))
-	for i, networkID := range options.NetworksID {
+	networks = make([]swarm.NetworkAttachmentConfig, len(options.Networks))
+	for i, network := range options.Networks {
+		aliases := make([]string, 0)
+		if network.Alias != "" {
+			aliases = append(aliases, network.Alias)
```

Comment:
why not just using `aliases := []string{network.Alias}` because we only have one alias all the time.

Reply:
I mean 1 alias at most. Something like this is fine:
```go
cfg := swarm.NetworkAttachmentConfig{
  Target:  network.ID,
}
if network.Alias != "" {
  cfg.Aliases = []string{network.Alias}
}
```

Reply:
done ;)